crates/astria-composer/src/executor/mod.rs

```diff
@@ -174,6 +174,14 @@ impl Executor {
     /// Create a future to submit a bundle to the sequencer.
     #[instrument(skip_all, fields(nonce.initial = %nonce))]
     fn submit_bundle(&self, nonce: u32, bundle: SizedBundle) -> Fuse<Instrumented<SubmitFut>> {
+        #[allow(clippy::cast_precision_loss)]
+        metrics::histogram!(crate::metrics_init::BUNDLES_OUTGOING_BYTES)
+            .record(bundle.get_size() as f64);
```

Comment:
These should be consistently emitted after a submission was successful

Reply:
cool, i ll emit them whenever there is a successful submission.